guide/designing-products/interoperability.md

```diff
@@ -4,32 +4,27 @@ title: Interoperability
 description: Designing for seamless experiences across bitcoin applications.
 nav_order: 7
 parent: Designing bitcoin products
-permalink: /guide/designing-products/wallet-interoperability/
+permalink: /guide/designing-products/interoperability/
+redirect_from: /guide/designing-products/wallet-interoperability/
 main_classes: -no-top-padding
-image: https://bitcoin.design/assets/images/guide/designing-products/wallet-interoperability/wallet-interoperability-preview.jpg
+image: https://bitcoin.design/assets/images/guide/designing-products/interoperability/wallet-interoperability-preview.jpg
 ---
 
 <!--
-
```

Comment:
Why delete the white space? Much harder to read now.

Reply:
That wasn't intentional, I overwrite the old MD file with the new and seems to have done that automatically, fixed.